python/cudf/cudf/tests/test_categorical.py

```diff
@@ -252,10 +252,10 @@ def test_cat_series_binop_error():
 @pytest.mark.parametrize("num_elements", [10, 100, 1000])
 def test_categorical_unique(num_elements):
     # create categorical series
-    np.random.seed(12)
+    rng = np.random.default_rng(seed=0)
```

Comment:
I suppose it doesn't really matter, but why the change of seed?

Reply:
It didn't actually matter keeping the seed same either:

```python
In [1]: import numpy as np

In [2]: np.random.seed(0)

In [3]: np.random.randint(0, 10)
Out[3]: 5

In [4]: np.random.default_rng(0).integers(0, 10).item()
Out[4]: 8
```